containers/scripts/run_hub.sh

```diff
@@ -40,6 +39,14 @@ if [ "$(/usr/lib/python3.9/site-packages/osh/hub/manage.py dumpdata scan.MockCon
         /usr/lib/python3.9/site-packages/osh/hub/scan/fixtures/initial_data.json
 fi
 
-
+if [[ -e /src ]]; then
+    echo "Woker should be started by podman-compose"
+else
+    # We are in the OpenShift deployment.
+    echo "Running resalloc-agent-spawner in the background. Logs would appear in /var/log/resalloc-agent-spawner/agent-spawner.log"
+    # TODO: This process should be watched over, probably at OpenShift level.
```

Comment:
@praiskup Any idea how to do that? Container should be restarted if this process dies.

Reply:
supervisord? or a separated "privileged" container with the access to the hub's database?

Reply:
@kdudka Do you have any recommendations here?

Reply:
Not really.

Reply:
We should probably do it through a liveness probe in OpenShift.

Reply:
https://pagure.io/fedora-infra/ansible/pull-request/1913

Reply:
So, things that go to stdout/stderr openshift captures and those get sent to our central log host, which in turn also sends them to RH's splunk. So, if these logs are good to keep, sending them out so openshift can capture them would result in them being in central logs/splunk too. ;)

Reply:
I have removed the redirection to `/dev/null` for now, but we need to improve logging in the agent spawner.